package.json

```diff
@@ -11,6 +11,7 @@
     "format": "prettier --write --ignore-path .gitignore src",
```

Comment:
It seems like AblyJS needs to be updated to fix the security issue found by `npm audit`.

Reply:
It actually forces a downgrade (the dependent libraries have not yet published a fix), becoming a breaking change for us :(

We'll need to monitor for the fix to be released for now.